apps/web/src/common/modules/navigations/gnb/modules/gnb-search-clone/GNBSearch.vue

```diff
@@ -420,13 +418,15 @@ watch(() => state.visible, async (visible) => {
         </g-n-b-search-dropdown>
         <div v-if="state.visible"
              class="background-block"
+             @click="hideSearchMenu"
```

Comment:
Suggestion: I think our convention of handler name should start with `handle~` prefix